### PR TITLE
Millestone 2 basic modules

### PR DESCRIPTION
**Pull request by FrancoRosa**
Hello TSEs,
In this milestone I manage to do:

- Create stateless react modules.
- Specify props types.
- Specify default props.
- Destructuring props.
- Make the basic look of the calculator project.

Thank you in advance for your kind feedback.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+);
+
+const App = () => (
+  <div className="app">
+    <Display />
+    <ButtonPanel />
+  </div>
+);
+
+export default App;
```
Comment (jofer86) [inline src/components/App.js]: For this par you need to divide your code into components, That is done in this file, but you should implement once component per file, and then have them come together. The Implementation is fine but your components need to have their own file. Other than that the project meets the criteria, except for that. So just give your components their own file and you're good to go.`

**Review by jofer86 — CHANGES_REQUESTED** [review]
## `const STATUS = "CHANGES REQUIRED"`⚠️ ⚠️ ⚠️ 
## :wave: :wave: Hello! I'm Jorge, and  I will be conducting this code review. Let me get a closer look 🔎 🔍 🔎 🔍 🕵🏼 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- Plsease read the comment below and do the necessary adjustments.
--------------------------------------------------------------------
**Rock on**  🤘🏼 
**_Jorge Fernando Rincón Figueroa._** 
Technical Support Engineer.

**Comment by FrancoRosa** [comment]
Hello Fernando,
I forgot to push once I divided the components on its file.
Now, every component lives on its own file
Thank you

**Review by shloch — APPROVED** [review]
## Hello @FrancoRosa 

Great great great.....keep up with the good work
accomplished

Now we can design
Happy coding

**Pull request merged**